public/app/core/logs_model.ts

```diff
@@ -19,25 +20,34 @@ export interface LogSearchMatch {
 export interface LogRow {
   key: string;
   entry: string;
+  labels: string;
   logLevel: LogLevel;
   timestamp: string;
   timeFromNow: string;
+  timeJs: number;
```

Comment:
A bit confusing name here with timestamp and timeJs, what is timeJs?

Reply:
Yeah, could use a better name. It's epoch millies. I need this to be the format that flot accepts as timestamp. Perhaps it's better to rename the other timestamp property to `timestampIso`.